pipelines/skylab/slideseq/test_inputs/Scientific/Puck_210817_11.mm10.json

```diff
@@ -0,0 +1,20 @@
+{
+  "SlideSeq.r1_fastq": [
```

Comment:
these are all in the hca-dcp-mint storage.

Reply:
ty @ekiernan! i believe @aawdeh will be replacing the hca-dcp-mint paths with broad-gtoc-test-storage paths in this ticket https://broadworkbench.atlassian.net/browse/PD-2092. I can update this branch once that gets merged into develop!

Reply:
The file paths are now updated.

Reply:
@aawdeh I am still seeing the paths point to hca-dcp-mint-test-data, do you need to push up another commit?

Reply:
@aawdeh looks good! 👍 i just needed to refresh